src/Core__Option.resi

```diff
@@ -96,8 +96,7 @@ Option.getUnsafe(None) // Raises an error
 external getUnsafe: option<'a> => 'a = "%identity"
 
 /**
-`mapWithDefault(opt, default, f)` applies `f` to `opt`, if `opt` is `None`, then
-`f` return `default`, otherwise return that value applied with `f`.
+`mapWithDefault(opt, default, f)` returns `Some(f(value))` if `opt` is `Some(value)`, otherwise `default`.
```

Comment:
Shouldn't this be "... returns f(value)"?

Reply:
Ah, yeah it should be. Good catch! Fixed in https://github.com/rescript-association/rescript-core/pull/70